src/app/QuickStarts/quickstarts/start-a-recording.tsx

```diff
@@ -111,15 +107,15 @@ There are two tabs within the Recordings page:
 
 [Active Recordings]{{highlight active-recordings-tab}} and [Archived Recordings]{{highlight archived-recordings-tab}}.
 
-Active recordings are recordings that only exist only within the target JVM. Archived recordings are recordings that have been saved from the target JVM and copied to Cryostat's storage volume.
+Active recordings are recordings that only exist only within the target JVM. Archived recordings are recordings that have been saved from the target JVM and copied to [APP]'s storage volume.
 
 We will start a recording while on the Active tab.
 
-1. Click [Create]{{highlight recordings-create-btn}} to go to the Custom Flight Recording Form.
+1. Click [Create]{{highlight recordings-create-btn}} to open the Custom Flight Recording Form.
```

Comment:
Please call out that the menu might be collapsed into a kebab (very likely since the quickstart panel takes up space).

Reply:
Also, bold **Custom Flight Recording Form**